app/models/spree/gift_card.rb

```diff
@@ -1,4 +1,4 @@
-require 'spree/core/validators/email'
+# require 'spree/core/validators/email'
```

Comment:
why we remove validators

Reply:
I commented on this line in an attempt to resolve the issue because I encountered the error 'cannot load such file -- spree/core/validators/email'